images/csi-node-driver-registrar.yml

```
@@ -8,7 +8,7 @@ content:
     git:
       branch:
         target: release-{MAJOR}.{MINOR}
-      url: https://github.com/openshift/csi-node-driver-registrar.git
+      url: https://github.com/openshift-priv/csi-node-driver-registrar.git
```

@shiywang e.g. this needs to be
```suggestion
      url: git@github.com:openshift-priv/csi-node-driver-registrar.git
```